r2dbc-mysql/src/main/java/JasyncClientConnection.kt

```diff
@@ -41,9 +76,14 @@ class JasyncClientConnection(private val jasyncConnection: JasyncConnection) : C
     }
 
     override fun setTransactionIsolationLevel(isolationLevel: IsolationLevel): Publisher<Void> {
+        this.isolationLevel = isolationLevel
```

Comment:
Setting should be deferred. See auto-commit comment.

Reply:
I couldn't find this information on the server status so kept it as is.